### ISSUE TITLE
`dev` Ruby version pages not working (404)

### ISSUE THREAD

**Issue by marcotc** [rubyapi/rubyapi#1189]
All `dev` pages I can find are currently returning 404, not found: https://rubyapi.org/dev/o/string

**Comment by natematykiewicz** [comment]
Looking into this locally, I see that I can't install Ruby 3.1.0 either.

The problem is this line:
https://github.com/rubyapi/rubyapi/blob/main/lib/ruby_type_signature_repository.rb#L60

```ruby
  def rbs_gem_path
    @rbs_path ||= Pathname(Dir.glob("#{@ruby_dir.join(RUBY_SRC_GEMS_FOLDER)}/rbs-*").find { |f| Pathname(f).directory? })
  end
```

`tmp/rubies/ruby-master/gems/rbs-*` does not exist on `ruby-master`. The `tmp/rubies/ruby-master/gems` folder only has 1 file in it: "bundled_gems", which has this in it:

```
minitest 5.14.0 https://github.com/seattlerb/minitest
power_assert 1.1.7 https://github.com/k-tsj/power_assert
rake 13.0.1 https://github.com/ruby/rake
test-unit 3.3.5 https://github.com/test-unit/test-unit
rexml 3.2.4 https://github.com/ruby/rexml
rss 0.2.9 https://github.com/ruby/rss
```

`tmp/rubies/ruby-3.1.0/gems` has these files:

- bundled_gems
- debug-1.4.0.gem
- matrix-0.4.2.gem
- minitest-5.15.0.gem
- net-ftp-0.1.3.gem
- net-imap-0.2.2.gem
- net-pop-0.1.1.gem
- net-smtp-0.3.1.gem
- power_assert-2.0.1.gem
- prime-0.1.2.gem
- rake-13.0.6.gem
- rbs-2.0.0.gem
- rexml-3.2.5.gem
- rss-0.2.9.gem
- test-unit-3.5.3.gem
- typeprof-0.21.1.gem

And its `bundled_gems` is:

```
# gem-name version-to-bundle repository-url [optional-commit-hash-to-test-or-defaults-to-v-version]
minitest 5.15.0 https://github.com/seattlerb/minitest
power_assert 2.0.1 https://github.com/ruby/power_assert
rake 13.0.6 https://github.com/ruby/rake
test-unit 3.5.3 https://github.com/test-unit/test-unit 3.5.3
rexml 3.2.5 https://github.com/ruby/rexml
rss 0.2.9 https://github.com/ruby/rss 0.2.9
net-ftp 0.1.3 https://github.com/ruby/net-ftp
net-imap 0.2.2 https://github.com/ruby/net-imap
net-pop 0.1.1 https://github.com/ruby/net-pop
net-smtp 0.3.1 https://github.com/ruby/net-smtp
matrix 0.4.2 https://github.com/ruby/matrix
prime 0.1.2 https://github.com/ruby/prime
rbs 2.0.0 https://github.com/ruby/rbs
typeprof 0.21.1 https://github.com/ruby/typeprof
debug 1.4.0 https://github.com/ruby/debug
```


Interestingly, in neither case do they have a `rbs-` file that's a directory though.

```
irb(main):001:0> path = Pathname('tmp/rubies/ruby-3.1.0/gems')
=> #<Pathname:tmp/rubies/ruby-3.1.0/gems>

path.directory?
=> true

path.glob('rbs-*')
=> [#<Pathname:tmp/rubies/ruby-3.1.0/gems/rbs-2.0.0.gem>]

irb(main):005:0> path.glob('rbs-*').first.directory?
=> false
```

I'm not really sure what the expectation is here.

**Comment by natematykiewicz** [comment]
Interesting. I just deleted my whole `tmp/rubies` folder. I can import Ruby 2.6.10, 2.7.6, 3.0.4, 3.1.2.

I tried a fresh download of 3.1.2 instead of the 3.1.0 that I had downloaded before.

```ruby
path = Pathname('tmp/rubies/ruby-3.1.2/gems')
=> #<Pathname:tmp/rubies/ruby-3.1.2/gems>

path.glob('rbs-*')
=> [#<Pathname:tmp/rubies/ruby-3.1.2/gems/rbs-2.1.0>, #<Pathname:tmp/rubies/ruby-3.1.2/gems/rbs-2.1.0.gem>]

path.glob('rbs-*').first.directory?
=> true
```

The `tmp/rubies/ruby-3.1.2/gems/rbs-2.1.0` folder seems to be a clone of https://github.com/ruby/rbs.

I also noticed my previous `ruby-master` folder was from January. It had told me `Found previously extracted download for dev, skipping`, and I hadn't thought anything of that. We should probably only do that if this isn't dev.

https://github.com/rubyapi/rubyapi/blob/main/lib/ruby_downloader.rb#L18
```ruby
if !release.dev? && already_fetched?
```

**Comment by natematykiewicz** [comment]
I just pushed up a PR to fix this. https://github.com/rubyapi/rubyapi/pull/1200

**Comment by natematykiewicz** [comment]
I merged #1200, but it seems like maybe the importer is a separate task you have to run manually.

**Comment by natematykiewicz** [comment]
I see that the `dev` docs are working. @colby-swandale must have re-ran the importer.
